src/customer/strategies/masterpass-customer-strategy.ts

```diff
@@ -0,0 +1,132 @@
+import { CheckoutStore, InternalCheckoutSelectors } from '../../checkout';
+import {
+    InvalidArgumentError,
+    MissingDataError,
+    MissingDataErrorType,
+    NotImplementedError,
+    NotInitializedError,
+    NotInitializedErrorType
+} from '../../common/error/errors';
+import { PaymentMethod, PaymentMethodActionCreator } from '../../payment';
+import { MasterpassScriptLoader } from '../../payment/strategies/masterpass';
+import { RemoteCheckoutActionCreator } from '../../remote-checkout';
+import CustomerCredentials from '../customer-credentials';
+import { CustomerInitializeOptions, CustomerRequestOptions } from '../customer-request-options';
+
+import CustomerStrategy from './customer-strategy';
+
+export default class MasterpassCustomerStrategy extends CustomerStrategy {
+    private _buttonId = 'mpbutton';
```

Comment:
So, the id of the button should always be `mpbutton` and cannot be changed? Shouldn't we let the user change it?